src/promptflow/promptflow/contracts/tool.py

```diff
@@ -298,6 +298,14 @@ def _deserialize_type(v):
             data.get("custom_type", []),
         )
 
+    def to_flow_input_definition(self):
+        # Used for eager flow to convert input definition to flow input definition.
```

Comment:
From concept level, do we always need this change? What it the definition have multiple types?

Reply:
The type isn't used in eager mode now, so it has no effect for executing, but it may cause problem in the future. And there is the same problem in normal flow, the tool resolver only respects the first one if multiple types are provided currently. So, we need to refine the code both in eager mode and normal flow.